apps/smp-server/Main.hs

```diff
@@ -1,13 +1,15 @@
+{-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE OverloadedStrings #-}
 
 module Main where
 
 import Control.Monad (when)
-import Data.Attoparsec.ByteString.Char8 (Parser)
```

Comment:
Could we gradually switch to importing specific identifiers / module as qualified, please? (in this case the former but I'm just saying as a general suggestion)
It's just so much nicer to support afterwards w/t having to rely on external tools to figure out what comes where from.